src/main/java/services/DatafeedEventsServiceV2.java

```diff
@@ -0,0 +1,102 @@
+package services;
+
+import clients.SymBotClient;
+import model.DatafeedEvent;
+import model.datafeed.DatafeedV2;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * DatafeedEventsServiceV2 providing services to bot for subscribing the datafeed version 2.
+ */
+class DatafeedEventsServiceV2 extends AbstractDatafeedEventsService {
```

Comment:
for compatibility I would suggest the same approach as for the client, that is to keep the DatafeedEventsService class and put inside it 
and how are we going to access the v2 service?
don't we need to change getDatafeedEventsService ?

Reply:
Actually, I'm using the same approach as for the client. You can see it here: https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/d8c88e4c3f1dbc445e092269e4259765de31beb2/src/main/java/services/DatafeedEventsService.java